Misc/NEWS.d/next/Core and Builtins/2021-03-21-18-51-30.bpo-43574.mteI-I.rst

```diff
@@ -0,0 +1,2 @@
+Restores previous list memory behavior where lists initialized from literals
```

Comment:
``list` objects don't overallocate when starting empty or are set to be empty. This effectively restores previous ``list`` memory behavior for lists initialized from literals.

Reply:
> ``list`

Minor typo - missing backtick in first \`\`list\`\`

Reply:
Added a commit to adopt this NEWS description (slightly modified)